LICENSE.md

```
@@ -0,0 +1,7 @@
+Copyright 2026 Comamoca
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)